astroquery/vo/utils.py

```diff
@@ -0,0 +1,258 @@
+#
+# Imports
+#
+
+import html # to unescape, which shouldn't be neccessary but currently is
+import io
+import numpy as np
+from astropy.table import Table
+
+#
+# Support for VOTABLEs as astropy tables
+#
+
+def astropy_table_from_votable_response(response):
+    """
+    Takes a VOTABLE response from a web service and returns an astropy table.
+
+    Parameters
+    ----------
+    response : requests.Response
+        Response whose contents are assumed to be a VOTABLE.
+
+    Returns
+    -------
+    astropy.table.Table
+        Astropy Table containing the data from the first TABLE in the VOTABLE.
+    """
+
+    # The astropy table reader would like a file-like object, so convert
+    # the response content a byte stream.  This assumes Python 3.x.
+    #
+    # (The reader also accepts just a string, but that seems to have two
+    # problems:  It looks for newlines to see if the string is itself a table,
+    # and we need to support unicode content.)
+    #Tracer()()
+    try:
+        file_like_content = io.BytesIO(response.content)
+    except IOError:
+        print("ERROR parsing response content? Got:\n    {}...".format(response.content[0:400]))
+        #  Should this return or raise?
+        raise
+    except Exception as e:
+        raise e
```

Comment:
this is redundant; just delete these two lines

Reply:
Addressed